cmd/snapshotgc/snapshotgc.go

```diff
@@ -188,8 +189,11 @@ func getSnapshotsForRemoval(
 	keptNonPrSnaps := 0
 
 	for _, snap := range snapshots {
+		snap := snap
 		label, found := snap.GetLabels()["pac.test.appstudio.openshift.io/event-type"]
-		if !found || label == "push" || label == "Push" {
+		isOverrideSnapshot := metadata.HasLabelWithValue(&snap, "test.appstudio.openshift.io/type", "override")
+		// override snapshot is covered by not found eveny-type label, but we still add it below 
```

Comment:
Maybe something like this would make it clearer?
```
override snapshot is not covered by the event-type label, but we still want to add it to the cleanup list
```

Reply:
Updated. Thanks.